option_go122.go

```diff
@@ -0,0 +1,22 @@
+//go:build go1.22
+// +build go1.22
```

Comment:
Did not know this syntax before.

It seems that it has been deprecated in go 1.17. So it would be unnecessary in this project (mo supports >= 1.18).